pkg/backend/httpstate/testdata/snapshot_test.json

```diff
@@ -0,0 +1,7 @@
+{
+  "req1": "{\"version\":3,\"deployment\":{\"manifest\":{\"time\":\"0001-01-01T00:00:00Z\",\"magic\":\"\",\"version\":\"\"},\"resources\":[\n{\"urn\":\"urn-1\",\"custom\":false,\"type\":\"\"}\n]}}",
```

Comment:
This is changing since indent scheme is changing, but it still passes the test that checks the logic.